src/marlin_stubs/host/M46.cpp

```diff
@@ -8,14 +8,13 @@ void GcodeSuite::M46() {
     netdev_get_ipv4_addresses(netdev_get_active_id(), &ethconfig);
     uint8_t *ipp = (uint8_t *)&ethconfig.addr_ip4;
     char ip4[17] = { 0 };
-    sprintf(ip4, "%u.%u.%u.%u\n", ipp[0], ipp[1], ipp[2], ipp[3]);
+    snprintf(ip4, 17, "%u.%u.%u.%u\n", ipp[0], ipp[1], ipp[2], ipp[3]);
```

Comment:
Small nitpick 😇: It would be better to either use constant or `sizeof ip4` instead of repeating that 17. The chance of the buffer size changing and getting out of sync is pretty low and they are close together, but it'd still be somewhat nicer.

Same goes for the MAC.

Reply:
Sure, no problem :) Should be fine now :)